test/test_numerizer_en.rb

```diff
@@ -101,7 +103,7 @@ def test_compatability
     assert_equal '1/2', Numerizer.numerize('1/2')
     assert_equal '05/06', Numerizer.numerize('05/06')
     assert_equal "3.5 hours", Numerizer.numerize("three and a half hours")
-    assert_equal "1/2 an hour", Numerizer.numerize("half an hour")
+    assert_equal "1/2 1 hour", Numerizer.numerize("half an hour")
```

Comment:
@jduff this might be an unintentional consequence of this change.

Reply:
hm, I don't really like that this changes, a fraction next to `1` is kind of weird. Can you look into it and see if there's a way to make this work without it affecting this test?

Reply:
To be honest, this problem existed earlier with "a".
```
assert_equal "1/2 a kilo", Numerizer.numerize("half a kilo")
```
![Screenshot 2021-02-18 at 11 30 57 AM](https://user-images.githubusercontent.com/16580576/108312460-c6fac880-71dc-11eb-83bc-a7b85f3a8364.png)